src/prisma/course.ts

```diff
@@ -0,0 +1,23 @@
+import { PrismaClient, Term } from '@prisma/client';
+export { findCourses, findMeetingTime };
+
+const prisma = new PrismaClient();
```

Comment:
There must be a better way of doing this, we should not have 3 prisma clients.

The index.ts file should be the module entrypoint

Reply:
Is it better to export the prisma from seed.ts?
Then we can use one client.

Reply:
Seed.ts does not run when the application is running. 

Just put prisma in the `index.ts` and use it via the other files in the module? Or is that not a thing?

Reply:
I changed the index.ts to user.ts.

Will create a new index.ts to include
export const prisma =new PrismaClient() 

Then other files will use this prisma from index.ts

Reply:
That should work.